tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1967,13 +2013,125 @@ def test_generate_with_prompt_ids(self):
         input_features = processor(input_speech, return_tensors="pt").input_features.to(torch_device)
 
         output_without_prompt = model.generate(input_features)
-        prompt_ids = processor.get_prompt_ids("Leighton")
+        prompt_ids = processor.get_prompt_ids("Leighton", return_tensors="pt").to(torch_device)
         output_with_prompt = model.generate(input_features, prompt_ids=prompt_ids)
 
         expected_without_prompt = "<|startoftranscript|><|en|><|transcribe|><|notimestamps|> He has grave doubts whether Sir Frederick Layton's work is really Greek after all and can discover in it but little of Rocky Ithaca.<|endoftext|>"
         expected_with_prompt = "<|startofprev|> Leighton<|startoftranscript|><|en|><|transcribe|><|notimestamps|> He has grave doubts whether Sir Frederick Leighton's work is really Greek after all and can discover in it but little of Rocky Ithaca.<|endoftext|>"
-        self.assertEqual(processor.decode(output_without_prompt[0]), expected_without_prompt)
-        self.assertEqual(processor.decode(output_with_prompt[0]), expected_with_prompt)
+
+        output_without_prompt = processor.decode(output_without_prompt[0])
+        output_with_prompt = processor.decode(output_with_prompt[0])
+
+        self.assertEqual(output_without_prompt, expected_without_prompt)
+        self.assertEqual(output_with_prompt, expected_with_prompt)
+
+    @slow
+    def test_language_detection(self):
```

Comment:
This test should be decorated with `@require_torchaudio`
```suggestion
    @require_torchaudio
    def test_language_detection(self):
```

Reply:
The whole test class has a `@require_torchaudio`